src/main/java/org/sagebionetworks/web/client/view/users/PasswordResetViewImpl.java

```diff
@@ -182,6 +187,11 @@ public void onClick(ClickEvent event) {
 		password1Field.getElement().setAttribute("placeholder", "Enter password");
 		password2Field.getElement().setAttribute("placeholder", "Confirm password");
 
+		emailAddressField.getElement().setAttribute(AUTOCOMPLETE_ATTRIBUTE, AUTOCOMPLETE_VALUE_USERNAME);
+		currentPasswordField.getElement().setAttribute(AUTOCOMPLETE_ATTRIBUTE, AUTOCOMPLETE_VALUE_CURRENT_PASSWORD);

```

Comment:
I couldn't figure out how to get the email address or current password fields to show in the PasswordReset place. Not sure if it's still possible to do so